packages/workers-shared/utils/helpers.ts

```diff
@@ -1,22 +1,11 @@
 import { getType } from "mime";
 
-/** normalises sep for windows, and encodes each segment */
-export const encodeFilePath = (filePath: string, sep: string) => {
-	const encodedPath = filePath
-		.split(sep)
-		.map((segment) => encodeURIComponent(segment))
-		.join("/");
+/** normalises sep for windows */
+export const normalizeFilePath = (filePath: string, sep: string) => {
```

Comment:
if this fn exists solely for the purpose of path normalization in Windows, would it make sense to then call it `normalizeFilePathInWin`?

Reply:
I'd push back on that name change. No need to leak the fact that this is needed for windows to the caller of the code.

Also there is the addition of a `/` prefix to all paths whether on Windows or not.